src/wasm-lib/kcl/src/source_range.rs

```diff
@@ -0,0 +1,132 @@
+use databake::*;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+use tower_lsp::lsp_types::{Position as LspPosition, Range as LspRange};
+
+/// Identifier of a source file.  Uses a u32 to keep the size small.
+#[derive(Debug, Default, Clone, Copy, PartialEq, Eq, Hash, Deserialize, Serialize, ts_rs::TS, JsonSchema, Bake)]
+#[cfg_attr(feature = "pyo3", pyo3::pyclass)]
```

Comment:
I think this was removed in your other PR.

Reply:
Yeah, I'll handle the rebasing depending on which lands first